cmd/sysl/datamodelview.go

```diff
@@ -210,6 +210,9 @@ func (v *DataModelView) drawTuple(
 				collectionString = fmt.Sprintf("+ %s : **Set <%s>**\n", attrName, path[0])
 				relation = `0..*`
 			default:
+				if attrType.GetTypeRef() == nil {
```

Comment:
I would change this to `case attrType.GetTypeRef() != nil:` and have the `default:` case just `continue`. This way it follows the same form as the rest of the switch statement.

Reply:
It is a hot fix as it is blocking team's work, and this change is a temp solution, will check root cause why attrType.GetTypeRef() return nil. So now it is a simple change.

Reply:
I'm sorry, but the change I am requesting is very simple - it is fewer lines of code than what you have done. Please keep the code clean.
```
case attrType.GetTypeRef() != nil:
	path = attrType.GetTypeRef().GetRef().Path
	collectionString = fmt.Sprintf("+ %s : **%s**\n", attrName, path[0])
	relation = `1..1 `
default:
	continue
```

Reply:
Fixed.